crates/rpc/rpc-convert/src/lib.rs

```diff
@@ -11,18 +11,16 @@
 #![cfg_attr(docsrs, feature(doc_cfg))]
 
 pub mod block;
-mod fees;
 pub mod receipt;
 mod rpc;
 pub mod transaction;
 
 pub use block::TryFromBlockResponse;
-pub use fees::{CallFees, CallFeesError};
 pub use receipt::TryFromReceiptResponse;
 pub use rpc::*;
 pub use transaction::{
-    EthTxEnvError, IntoRpcTx, RpcConvert, RpcConverter, TransactionConversionError,
-    TryFromTransactionResponse, TryIntoSimTx, TxInfoMapper,
+    RpcConvert, RpcConverter, TransactionConversionError, TryFromTransactionResponse, TryIntoSimTx,
+    TxInfoMapper,
```

Comment:
can we also re-export the types we moved to alloy-evm here?

then this becomes non-breaking